test/pytest/test_multiout_network.py

```diff
@@ -19,25 +19,40 @@ def model():
     return model
 
 
+@pytest.fixture(scope='module')
+def model2():
+    in1 = keras.layers.Input(shape=(24, 8))
+    in2 = keras.layers.Input(shape=(16))
+    out1 = keras.layers.Conv1D(1, 3)(in1)
+    out1 = keras.layers.Flatten()(out1)
```

Comment:
I would prefer if the variables names weren't reused since initially it confused me, but it really is a minor point, so if am fine leaving it as is.

Reply:
It is actually a rather common practice with keras functional api to reuse same variable for representing the same data stream, see example at https://www.tensorflow.org/guide/keras/functional_api#a_toy_resnet_model